packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -294,6 +301,11 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, Semver {
         l2BlockNumber_ = _getArgUint256(0x20);
     }
 
+    /// @inheritdoc IFaultDisputeGame
+    function l1BlockNumber() public pure returns (uint256 l1BlockNumber_) {
```

Comment:
What is the set of valid l1 block numbers given a particular dispute game? It just impacts how the game is played correct?

Reply:
This L1 block number represents the block that the creator of the game begins running the program from (see `op-program`'s `--l1.head` flag), so there's no verification to perform on it. If a game is created with a L1 block number that is invalid for the dispute, another game can be created in its place due to the `l1BlockNumber` being a part of `extraData`, which is included in the construction of the UUID hash.

Reply:
Can an invalid l1 block number challenge agents to respond and subsequently result in this invalid dispute game to resolve incorrectly since the "honest" actors clocks run out?

Reply:
Answered offline